src/XMakeBuildEngine/Evaluation/ProjectParser.cs

```diff
@@ -194,32 +194,23 @@ private void ParseProjectElement(XmlElementWithLocation element)
                 // TODO: multiples
                 //  TODO: Don't null ref if MSBuildSDKsPath isn't defined
 
-                // TODO: paths should just be Sdk.props/targets; Sdk-aware imports should do the rest of the path.
-                var initialImportPath = Path.Combine(Environment.GetEnvironmentVariable("MSBuildSDKsPath"),
-                    element.GetAttribute("Sdk"), "Sdk", "Sdk.props");
-                var finalImportPath = Path.Combine(Environment.GetEnvironmentVariable("MSBuildSDKsPath"),
-                    element.GetAttribute("Sdk"), "Sdk", "Sdk.targets");
-
-                if (File.Exists(initialImportPath))
+                //  TODO: Add file existence conditions to the imports
+                //if (File.Exists(initialImportPath))
```

Comment:
Hmm, this is actually really hard if we go down this path, isn't it? We don't have a way to condition the import on existence, because we don't actually know the path to plug into an `Exists()` check.

Can you think of another way besides adding an `IfExists="true"` attribute to `<Import>`?